internal/html-parser/index.ts

```diff
@@ -336,78 +395,115 @@ function parseTag(parser: HTMLParser): HTMLElement {
 	);
 }
 
+function parserDoctype(parser: HTMLParser): HTMLDoctypeTag | undefined {
```

Comment:
```suggestion
function parseDoctype(parser: HTMLParser): HTMLDoctypeTag | undefined {
```
Maybe it should be `parseDoctype` ? :)